testonly/hammer/hammer.go

```diff
@@ -22,6 +22,7 @@ import (
 	"strconv"
 	"strings"
 	"sync"
+	"syscall"
```

Comment:
[Library docs](https://golang.org/pkg/syscall/) say: "Deprecated: this package is locked down. Callers should use the corresponding package in the golang.org/x/sys repository instead."

Reply:
Thanks. Migrated